tests/system/Helpers/TextHelperTest.php

```diff
@@ -263,7 +263,14 @@ public function testCensoredWordsNonReplacement(): void
 
     public function testHighlightCode(): void
     {
-        $expect = "<code><span style=\"color: #000000\">\n<span style=\"color: #0000BB\">&lt;?php&nbsp;var_dump</span><span style=\"color: #007700\">(</span><span style=\"color: #0000BB\">\$this</span><span style=\"color: #007700\">);&nbsp;</span><span style=\"color: #0000BB\">?&gt;&nbsp;</span>\n</span>\n</code>";
+        // PHP 8.3 changes the output.
+        if (version_compare(PHP_VERSION, '8.3', '<')) {
```

Comment:
For clarity can we use PHP_VERSION_ID instead?

Reply:
Nice! Refactored.